{"changes": {"content": {"new": "Hi,\r\nI am attempting to catkin_make install in the custom controller tutorial.\r\nIt gives me this error.\r\nphil@phil-ThinkPad-T460:~/subt_seed_ws$ catkin_make install\r\nBase path: /home/phil/subt_seed_ws\r\nSource space: /home/phil/subt_seed_ws/src\r\nBuild space: /home/phil/subt_seed_ws/build\r\nDevel space: /home/phil/subt_seed_ws/devel\r\nInstall space: /home/phil/subt_seed_ws/install\r\n####\r\n#### Running command: \"cmake /home/phil/subt_seed_ws/src -DCATKIN_DEVEL_PREFIX=/home/phil/subt_seed_ws/devel -DCMAKE_INSTALL_PREFIX=/home/phil/subt_seed_ws/install -G Unix Makefiles\" in \"/home/phil/subt_seed_ws/build\"\r\n####\r\n-- Using CATKIN_DEVEL_PREFIX: /home/phil/subt_seed_ws/devel\r\n-- Using CMAKE_PREFIX_PATH: /home/phil/subt_seed_ws/devel;/opt/ros/melodic\r\n-- This workspace overlays: /home/phil/subt_seed_ws/devel;/opt/ros/melodic\r\n-- Found PythonInterp: /usr/bin/python2 (found suitable version \"2.7.15\", minimum required is \"2\") \r\n-- Using PYTHON_EXECUTABLE: /usr/bin/python2\r\n-- Using Debian Python package layout\r\n-- Using empy: /usr/bin/empy\r\n-- Using CATKIN_ENABLE_TESTING: ON\r\n-- Call enable_testing()\r\n-- Using CATKIN_TEST_RESULTS_DIR: /home/phil/subt_seed_ws/build/test_results\r\n-- Found gmock sources under '/usr/src/googletest': gmock will be built\r\n-- Found PythonInterp: /usr/bin/python2 (found version \"2.7.15\") \r\n-- Found gtest sources under '/usr/src/googletest': gtests will be built\r\n-- Using Python nosetests: /usr/bin/nosetests-2.7\r\n-- catkin 0.7.14\r\n-- BUILD_SHARED_LIBS is on\r\n-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\r\n-- ~~  traversing 1 packages in topological order:\r\n-- ~~  - subt_seed\r\n-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\r\n-- +++ processing catkin package: 'subt_seed'\r\n-- ==> add_subdirectory(subt_seed2)\r\n-- Using these message generators: gencpp;geneus;genlisp;gennodejs;genpy\r\n-- Could NOT find subt_gazebo (missing: subt_gazebo_DIR)\r\n-- Could not find the required component 'subt_gazebo'. The following CMake error indicates that you either need to install the package with the same name or change your environment so that it can be found.\r\nCMake Error at /opt/ros/melodic/share/catkin/cmake/catkinConfig.cmake:83 (find_package):\r\n  Could not find a package configuration file provided by \"subt_gazebo\" with\r\n  any of the following names:\r\n\r\n    subt_gazeboConfig.cmake\r\n    subt_gazebo-config.cmake\r\n\r\n  Add the installation prefix of \"subt_gazebo\" to CMAKE_PREFIX_PATH or set\r\n  \"subt_gazebo_DIR\" to a directory containing one of the above files.  If\r\n  \"subt_gazebo\" provides a separate development package or SDK, be sure it\r\n  has been installed.\r\nCall Stack (most recent call first):\r\n  subt_seed2/CMakeLists.txt:14 (find_package)\r\n\r\n\r\n-- Configuring incomplete, errors occurred!\r\nSee also \"/home/phil/subt_seed_ws/build/CMakeFiles/CMakeOutput.log\".\r\nSee also \"/home/phil/subt_seed_ws/build/CMakeFiles/CMakeError.log\".", "old": "Hi,\r\nI am attempting to catkin_make install in the custom controller tutorial.\r\nI gives me this error.\r\nphil@phil-ThinkPad-T460:~/subt_seed_ws$ catkin_make install\r\nBase path: /home/phil/subt_seed_ws\r\nSource space: /home/phil/subt_seed_ws/src\r\nBuild space: /home/phil/subt_seed_ws/build\r\nDevel space: /home/phil/subt_seed_ws/devel\r\nInstall space: /home/phil/subt_seed_ws/install\r\n####\r\n#### Running command: \"cmake /home/phil/subt_seed_ws/src -DCATKIN_DEVEL_PREFIX=/home/phil/subt_seed_ws/devel -DCMAKE_INSTALL_PREFIX=/home/phil/subt_seed_ws/install -G Unix Makefiles\" in \"/home/phil/subt_seed_ws/build\"\r\n####\r\n-- Using CATKIN_DEVEL_PREFIX: /home/phil/subt_seed_ws/devel\r\n-- Using CMAKE_PREFIX_PATH: /home/phil/subt_seed_ws/devel;/opt/ros/melodic\r\n-- This workspace overlays: /home/phil/subt_seed_ws/devel;/opt/ros/melodic\r\n-- Found PythonInterp: /usr/bin/python2 (found suitable version \"2.7.15\", minimum required is \"2\") \r\n-- Using PYTHON_EXECUTABLE: /usr/bin/python2\r\n-- Using Debian Python package layout\r\n-- Using empy: /usr/bin/empy\r\n-- Using CATKIN_ENABLE_TESTING: ON\r\n-- Call enable_testing()\r\n-- Using CATKIN_TEST_RESULTS_DIR: /home/phil/subt_seed_ws/build/test_results\r\n-- Found gmock sources under '/usr/src/googletest': gmock will be built\r\n-- Found PythonInterp: /usr/bin/python2 (found version \"2.7.15\") \r\n-- Found gtest sources under '/usr/src/googletest': gtests will be built\r\n-- Using Python nosetests: /usr/bin/nosetests-2.7\r\n-- catkin 0.7.14\r\n-- BUILD_SHARED_LIBS is on\r\n-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\r\n-- ~~  traversing 1 packages in topological order:\r\n-- ~~  - subt_seed\r\n-- ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\r\n-- +++ processing catkin package: 'subt_seed'\r\n-- ==> add_subdirectory(subt_seed2)\r\n-- Using these message generators: gencpp;geneus;genlisp;gennodejs;genpy\r\n-- Could NOT find subt_gazebo (missing: subt_gazebo_DIR)\r\n-- Could not find the required component 'subt_gazebo'. The following CMake error indicates that you either need to install the package with the same name or change your environment so that it can be found.\r\nCMake Error at /opt/ros/melodic/share/catkin/cmake/catkinConfig.cmake:83 (find_package):\r\n  Could not find a package configuration file provided by \"subt_gazebo\" with\r\n  any of the following names:\r\n\r\n    subt_gazeboConfig.cmake\r\n    subt_gazebo-config.cmake\r\n\r\n  Add the installation prefix of \"subt_gazebo\" to CMAKE_PREFIX_PATH or set\r\n  \"subt_gazebo_DIR\" to a directory containing one of the above files.  If\r\n  \"subt_gazebo\" provides a separate development package or SDK, be sure it\r\n  has been installed.\r\nCall Stack (most recent call first):\r\n  subt_seed2/CMakeLists.txt:14 (find_package)\r\n\r\n\r\n-- Configuring incomplete, errors occurred!\r\nSee also \"/home/phil/subt_seed_ws/build/CMakeFiles/CMakeOutput.log\".\r\nSee also \"/home/phil/subt_seed_ws/build/CMakeFiles/CMakeError.log\"."}}, "links": {"self": {"href": "data/repositories/osrf/subt/issues/31/changes/49263132.json"}, "html": {"href": "#!/osrf/subt/issues/31#comment-49263132"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/subt/issues/31.json"}}, "type": "issue", "id": 31, "repository": {"links": {"self": {"href": "data/repositories/osrf/subt.json"}, "html": {"href": "#!/osrf/subt"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3c95f6ad-c304-407c-b838-09597d836552}ts=2272898"}}, "type": "repository", "name": "subt", "full_name": "osrf/subt", "uuid": "{3c95f6ad-c304-407c-b838-09597d836552}"}, "title": "custom controller error"}, "created_on": "2018-12-03T19:23:39.038567+00:00", "user": {"display_name": "Phillip Seaton", "uuid": "{1ebcff26-ac11-4ccb-ab8f-4b832df81c63}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B1ebcff26-ac11-4ccb-ab8f-4b832df81c63%7D"}, "html": {"href": "https://bitbucket.org/%7B1ebcff26-ac11-4ccb-ab8f-4b832df81c63%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/f5c6611802cb85a24c589c0f31fcb389d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsPS-6.png"}}, "nickname": "pbs5kx", "type": "user", "account_id": "5c055cebe578fa1435db1941"}, "message": {"raw": null, "markup": "markdown", "html": "", "type": "rendered"}, "type": "issue_change", "id": 49263132}